engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeErrorsTest.scala

```diff
@@ -175,7 +175,7 @@ class RuntimeErrorsTest
         contextId,
         xId,
         Api.ExpressionUpdate.Payload.Panic(
-          "Compile error: The name `undefined` could not be found.",
+          "Compile_Error.Error",
           Seq(xId)
```

Comment:
I'm slightly worried about losing these.

Are we sure that everything is wired in correctly that in production we will get the human readable errors? Getting `Compile_Error.Error` will be bad.

Cannot we import the part of stdlib responsible for the `to_display_text` here?

I think that ensuring that the errors are sensible in production is pretty important, so I'd advocate for just importing the full stdlib here. The compilation time is now much shorter, I think we can pay the extra 30s on the CI for ensuring that errors are correct.

Reply:
Especially as, we were quite often having bugs in our `to_display_text`. Nothing better than your code erroring when trying to display the error, so that you can error while you error and do not know the true error.

I think it's really good to have them tested.

Reply:
See my comment in `Important Notes` section.

If something is not a builtin and you don't import the corresponding type from stdlib then how is it supposed to know about that method? All it knows is just builtins.
Let's be honest, IDE will always import `Standard.Base.Main` which should and does import those, so you won't see those in real life. Those test cases often don't import stdlib, for brevity, hence the error. That is also why I added sometimes a `pretty print` version, even if it is not really necessary.

A similar problem was with `Any.Any` during the equals work. That's why all our stdlib and tests are now _polluted_ with that import.

Reply:
re 2: but I didn't ignore testing the human-readable version of errors. See for example `RuntimeServerTest`. I'm fine to add a similar version here, in `RuntimeErrorsTes`, though the whole calculation of location is rather cumbersome. That's why I skipped it in this case.

Reply:
Oh! Sorry, the diff was very large there and I didn't see this part. My bad. Looks all good then.

Reply:
Added tests with human-readable text as well.